### PR TITLE
Add metrics collection and metrics-based dashboards info to RabbitMQ

### PR DESCRIPTION
**Pull request by yasar-sumologic**
…Otel

## Purpose of this pull request

Changes to rabbitmq-otel app doc after changes in it to add metric collection and metric-based dashboards

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 1]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/containers-orchestration/rabbitmq.png')} alt="icon" width="45"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-The [RabbitMQ](https://www.rabbitmq.com/getstarted.html) app is a unified log app. Preconfigured dashboards provide insight into error logs. RabbitMQ logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The [RabbitMQ](https://www.rabbitmq.com/getstarted.html) app is a unified log app. Preconfigured dashboards provide insight into error logs and metrics. RabbitMQ logs and metrics are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:14]:
```suggestion
The Sumo Logic app for [RabbitMQ](https://www.rabbitmq.com/getstarted.html) is a unified log solution that includes preconfigured dashboards offering insights into error logs and metrics. RabbitMQ logs and metrics are transmitted to Sumo Logic via the OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 2]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
...
 ## Fields creation in Sumo Logic for RabbitMQ
 
 Following are the [Fields](/docs/manage/fields/) which will be created as part of RabbitMQ App install if not already present.
 
-* `messaging.cluster.name`. User configured. Specify the user-friendly cluster name which RabbitMQ belongs to.
-* `sumo.datasource`. Has fixed value of **rabbitmq**.
+* **`sumo.datasource`**. Fixed value of **rabbitmq**.
+* **`messaging.system`**. Fixed value of **rabbitmq**.
+* **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the rabbitmq env resides, such as `dev`, `prod`, or `qa`.
+* **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your Rabbitmq cluster. This cluster name will be shown in the Sumo Logic dashboards.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:25]:
```suggestion
* **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your RabbitMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 3]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
...
+* **`messaging.node.name`**. Includes the value of the hostname of the machine which is being monitored.
 
 ## Prerequisites
 
+### For metrics collection
+
+This RabbitMQ receiver fetches stats from a RabbitMQ node using the [RabbitMQ Management Plugin](https://www.rabbitmq.com/docs/management) and converts them to OpenTelemetry metrics
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:32]:
```suggestion
This RabbitMQ receiver fetches stats from a RabbitMQ node using the [RabbitMQ Management Plugin](https://www.rabbitmq.com/docs/management) and converts them to OpenTelemetry metrics.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 4]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
...
-2. **Configure RabbitMQ to write log lines to a local file**. By default, RabbitMQ logs are stored in `/var/log/rabbitmq/rabbit@<hostname>.log`. The default directory for log files is listed in the `RabbitMQ.conf` file. To configure the log output destination to a log file, use one of the following settings, either in the [configuration file](https://www.rabbitmq.com/logging.html). Edit or create `/etc/rabbitmq/rabbitmq.conf` file config:
+
+1. RabbitMQ logs have six levels of verbosity: debug, info, warning, error, critical, none. For details please visit this [page](https://www.rabbitmq.com/logging.html#log-levels). For the dashboards to work properly, log level needs to be set to **debug**. Default log level is **info**. All logging settings are located in [RabbitMQ.conf](https://www.rabbitmq.com/logging.html).
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:41]:
```suggestion
1. RabbitMQ logs have six levels of verbosity: debug, info, warning, error, critical, and none. For details, visit the [RabbitMQ documentation](https://www.rabbitmq.com/logging.html#log-levels). For the dashboards to work properly, the log level needs to be set to **debug**. Default log level is **info**. All logging settings are located in [RabbitMQ.conf](https://www.rabbitmq.com/logging.html).
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 5]
```diff
@@ -74,9 +84,18 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
...
+
+In this step, you will configure the yaml file required for RabbitMQ Collection.
 
-The files are typically located in `/var/log/rabbitmq/rabbit@<hostname>.log`. You can add any custom fields which you want to tag along with the data ingested in sumo. Click on the **Download YAML File** button to get the yaml file.
+Below are the inputs required:
+
+- **`endpoint (no default)`**. The hostname and port of the RabbitMQ instance, separated by a colon. (For example: `localhost:15672`.)
+- **RabbitMQ logs Path**. Enter the path to the log file for your RabbitMQ instance.
+
+The log file path configured to capture rabbitmq logs must be given here. The files are typically located in `/var/log/rabbitmq/rabbit@<hostname>.log`. If you are using a customized path, check the [`rabbitmq.conf`](https://www.rabbitmq.com/logging.html) file for this information.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:96]:
```suggestion
The log file path configured to capture RabbitMQ logs must be given here. The files are typically located in `/var/log/rabbitmq/rabbit@<hostname>.log`. If you are using a customized path, check the [`rabbitmq.conf`](https://www.rabbitmq.com/logging.html) file for this information.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 6]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/containers-orchestration/rabbitmq.png')} alt="icon" width="45"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-The [RabbitMQ](https://www.rabbitmq.com/getstarted.html) app is a unified log app. Preconfigured dashboards provide insight into error logs. RabbitMQ logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The Sumo Logic app for [RabbitMQ](https://www.rabbitmq.com/getstarted.html) is a unified log solution that includes preconfigured dashboards offering insights into error logs and metrics. RabbitMQ logs and metrics are transmitted to Sumo Logic via the OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```
Comment (sumoanema) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:14]: this is incorrect. Logs are through filelog receiver and metric will be through rabbit mq receiver

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 7]
```diff
@@ -11,25 +11,35 @@ import TabItem from '@theme/TabItem';
...
+* **`messaging.system`**. Fixed value of **rabbitmq**.
+* **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the rabbitmq env resides, such as `dev`, `prod`, or `qa`.
+* **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your RabbitMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* **`messaging.node.name`**. Includes the value of the hostname of the machine which is being monitored.
 
 ## Prerequisites
 
+### For metrics collection
+
+This RabbitMQ receiver fetches stats from a RabbitMQ node using the [RabbitMQ Management Plugin](https://www.rabbitmq.com/docs/management) and converts them to OpenTelemetry metrics.
```
Comment (sumoanema) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:32]: Can you copy the whole prerequisite section from rabbitmq receiver doc : https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/rabbitmqreceiver#prerequisites

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 8]
```diff
@@ -74,9 +84,18 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
...
 
-In this step, we will be configuring the yaml file required for RabbitMQ Collection. Path of the log file configured to capture RabbitMQ logs is needed to be given here.
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, you will configure the yaml file required for RabbitMQ Collection.
 
-The files are typically located in `/var/log/rabbitmq/rabbit@<hostname>.log`. You can add any custom fields which you want to tag along with the data ingested in sumo. Click on the **Download YAML File** button to get the yaml file.
+Below are the inputs required:
+
+- **`endpoint (no default)`**. The hostname and port of the RabbitMQ instance, separated by a colon. (For example: `localhost:15672`.)
```
Comment (sumoanema) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:93]: I see in the rabbitmq doc that username and password are required parameters. But here they are not listed. Are we not using them in them in the input configuration ?

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 6 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it.

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 7 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: updated it

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 8 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:96]: Added username and password as required parameters.

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 5 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it.

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 4 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 3 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it.

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 2 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it.

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md (hunk 1 above)
Comment (yasar-sumologic) [reply docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md]: fixed it

**Comment by yasar-sumologic** [comment]
Hi @jpipkin1 / @JV0812 / @mafsumo / @kimsauce 
I've addressed your feedback on PR and believe it's ready for re-review; could you please take another look?

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 9]
```diff
@@ -11,25 +11,38 @@ import TabItem from '@theme/TabItem';
...
+* **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the rabbitmq env resides, such as `dev`, `prod`, or `qa`.
+* **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your RabbitMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* **`messaging.node.name`**. Includes the value of the hostname of the machine which is being monitored.
 
 ## Prerequisites
 
+### For metrics collection
+
+The RabbitMQ receiver supports RabbitMQ versions `3.8` and `3.9`.
+The RabbitMQ Management Plugin must be enabled by following the [official instructions](https://www.rabbitmq.com/management.html#getting-started).
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:33]:
```suggestion
The RabbitMQ management plugin must be enabled by following the [official instructions](https://www.rabbitmq.com/management.html#getting-started).
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 10]
```diff
@@ -11,25 +11,38 @@ import TabItem from '@theme/TabItem';
...
+* **`messaging.system`**. Fixed value of **rabbitmq**.
+* **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the rabbitmq env resides, such as `dev`, `prod`, or `qa`.
+* **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your RabbitMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* **`messaging.node.name`**. Includes the value of the hostname of the machine which is being monitored.
 
 ## Prerequisites
 
+### For metrics collection
+
+The RabbitMQ receiver supports RabbitMQ versions `3.8` and `3.9`.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:32]:
```suggestion
:::note
The RabbitMQ receiver supports RabbitMQ versions `3.8` and `3.9`.
:::
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 11]
```diff
@@ -11,25 +11,38 @@ import TabItem from '@theme/TabItem';
...
+The RabbitMQ Management Plugin must be enabled by following the [official instructions](https://www.rabbitmq.com/management.html#getting-started).
+
+Also, a user with at least [monitoring](https://www.rabbitmq.com/management.html#permissions) level permissions must be used for monitoring.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:35]:
```suggestion
Also, make sure you at least have the [monitoring](https://www.rabbitmq.com/management.html#permissions) level permissions to monitor the metrics.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 12]
```diff
@@ -11,25 +11,38 @@ import TabItem from '@theme/TabItem';
...
-2. **Configure RabbitMQ to write log lines to a local file**. By default, RabbitMQ logs are stored in `/var/log/rabbitmq/rabbit@<hostname>.log`. The default directory for log files is listed in the `RabbitMQ.conf` file. To configure the log output destination to a log file, use one of the following settings, either in the [configuration file](https://www.rabbitmq.com/logging.html). Edit or create `/etc/rabbitmq/rabbitmq.conf` file config:
+
+1. RabbitMQ logs have six levels of verbosity: debug, info, warning, error, critical, and none. For details, visit the [RabbitMQ documentation](https://www.rabbitmq.com/logging.html#log-levels). For the dashboards to work properly, the log level needs to be set to **debug**. Default log level is **info**. All logging settings are located in [RabbitMQ.conf](https://www.rabbitmq.com/logging.html).
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:44]:
```suggestion
1. **RabbitMQ logs have six levels of verbosity**. Debug, info, warning, error, critical, and none. For more information, refer to the [RabbitMQ documentation](https://www.rabbitmq.com/logging.html#log-levels). To make sure the dashboard work properly, the log level needs to be set to **debug**. Default log level is **info**. All logging settings are located in [RabbitMQ.conf](https://www.rabbitmq.com/logging.html).
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 13]
```diff
@@ -74,9 +87,22 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 ### Step 2: Configure integration
 
-In this step, we will be configuring the yaml file required for RabbitMQ Collection. Path of the log file configured to capture RabbitMQ logs is needed to be given here.
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, you will configure the yaml file required for RabbitMQ Collection.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:92]:
```suggestion
In this step, you will configure the yaml file required for RabbitMQ collection.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 14]
```diff
@@ -173,10 +201,26 @@ This sample Query is from the **RabbitMQ - Logs dashboard** > **Events** by Seve
 | parse "* * [*]" as date,time,severity | count by severity
 ```
 
+### Metrics
+
+This sample query is from the **RabbitMQ - Metrics dashboard** > **Average Number of Consumers**  panel.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:206]:
```suggestion
This sample query is from the **Average Number of Consumers** panel in **RabbitMQ - Metrics** dashboard.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 15]
```diff
@@ -173,10 +201,26 @@ This sample Query is from the **RabbitMQ - Logs dashboard** > **Events** by Seve
...
+### Logs
+
+The **RabbitMQ - Logs** dashboard gives you an at-a-glance view of Error messages, error by severity, top and last 10 errors, Broker and Event Start/Add log messages.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/RabbitMq-OpenTelemetry/RabbitMQ-Logs.png' alt="RabbitMQ Logs dashboards" />
+
+### Metrics
 
-The **RabbitMQ - Overview** dashboard gives you an at-a-glance view of Error messages, error by severity, top and last 10 errors, Broker and Event Start/Add log messages.
+The **RabbitMQ - Metrics** dashboard gives you an at-a-glance view of your RabbitMQ deployment across brokers, queue, exchange, consumer, messages.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:226]:
```suggestion
The **RabbitMQ - Metrics** dashboard gives you an at-a-glance view of your RabbitMQ deployment across brokers, queue, exchange, consumer, and messages.
```

File: docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md [hunk 16]
```diff
@@ -173,10 +201,26 @@ This sample Query is from the **RabbitMQ - Logs dashboard** > **Events** by Seve
...
+| sum
+```
+
+
 ## Viewing RabbitMQ Dashboards
 
-### Overview
+### Logs
+
+The **RabbitMQ - Logs** dashboard gives you an at-a-glance view of Error messages, error by severity, top and last 10 errors, Broker and Event Start/Add log messages.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md:220]:
```suggestion
The **RabbitMQ - Logs** dashboard gives you an at-a-glance view of error messages, error by severity, top and last 10 errors, broker, and Event Start/Add log messages.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by jpipkin1** [comment]
@yasar-sumologic, can we publish this PR now, or do we need to wait for a release?

**Pull request merged**